src/components/views/settings/encryption/ChangeRecoveryKey.tsx

```diff
@@ -32,9 +32,15 @@ import { withSecretStorageKeyCache } from "../../../../SecurityManager";
  * - `inform_user`: The user is informed about the recovery key.
  * - `save_key_setup_flow`: The user is asked to save the new recovery key during the setup flow.
  * - `save_key_change_flow`: The user is asked to save the new recovery key during the change key flow.
- * - `confirm`: The user is asked to confirm the new recovery key.
+ * - `confirm_key_setup_flow`: The user is asked to confirm the new recovery key during the set up flow.
+ * - `confirm_key_change_flow`: The user is asked to confirm the new recovery key during the change key flow.
```

Comment:
I have to say, I wonder if it wouldn't be simpler to just have three states (`inform_user`, `save_key`, `confirm_key`) and distinguish between the two flows by looking at the `userHasRecoveryKey` prop.

On the other hand, we should probably just get on and land this rather than messing with it forever. A thought for the future though.